platform/admin/docs/src/main/resources/_contents/managing-contents.adoc

```diff
@@ -246,7 +243,7 @@ To do so, follow these instructions:
 
 . Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
```

Comment:
Fixed. Also, moving this entire section to the system managing docs, under Importing configurations.